scripts/msct_image.py

```diff
@@ -216,6 +240,9 @@ def __init__(self, im, axis="IS"):
         if dim_nr == -1:
             raise ValueError()
 
+        # SCT convention
```

Comment:
(sorry i can't add comment on a line that was not modified).
comment for 253: i would replace "a multi-dimensional slice" by "a 2D slice", because this isn't it what it does? otherwise i would not call it a "slice" (which by definition is 2D).

Reply:
A **multi-dimensional slice** is the conventional term for objects that are an N-tuple or `None` or `slice` objects. 
When you say `arr[1,:,2]` this is multi-dimensional slicing of `arr`.